core/src/themes/ionic.globals.ionic.scss

```diff
@@ -0,0 +1,209 @@
+// Global Font Functions
+@import "../themes/ionic.functions.font";
+
+$ionic-display-m:         dynamic-font(36px);
+$ionic-display-s:         dynamic-font(32px);
+
+$ionic-h1-font-size:      dynamic-font(28px) !default;
+$ionic-h2-font-size:      dynamic-font(26px) !default;
+$ionic-h3-font-size:      dynamic-font(24px) !default;
+$ionic-h4-font-size:      dynamic-font(22px) !default;
+$ionic-h5-font-size:      dynamic-font(20px) !default;
+$ionic-h6-font-size:      dynamic-font(18px) !default;
+
+$ionic-font-size-s:       dynamic-font(12px);
+$ionic-font-size-base:    dynamic-font(14px);
+$ionic-font-size-l:       dynamic-font(16px);
+
+:root {
+  /* Font Family */
+  --ionic-font-family: "Inter", sans-serif;
+
+  /* Font sizes */
+  --ionic-font-size-s: #{$ionic-font-size-s};
+  --ionic-font-size-base: #{$ionic-font-size-base};
+  --ionic-font-size-l: #{$ionic-font-size-l};
+
+  /* Font size displays */
+  --ionic-display-s: #{$ionic-display-s};
+  --ionic-display-m: #{$ionic-display-m};
+  --ionic-line-height-display-s: 44px;
```

Comment:
I noticed that some variables are using other variables or they'll be using the value. What determines when you use what?

Reply:
Related to that use cases, is because of the dynamic-font function usage. We need to make the calculations on Sass variable and set it to the css variable. But, based on your feedback, we will not use the dynamic-font for now, so I will remove all of this.